modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -155,10 +158,15 @@ export default class LargeVideoManager {
 
             // Make sure no notification about remote failure is shown as
             // its UI conflicts with the one for local connection interrupted.
+            let isConnected = APP.conference.isConnectionInterrupted()
+                                || (!isHavingConnectivityIssues && isInLastN);
+
             this.updateParticipantConnStatusIndication(
                     id,
-                    APP.conference.isConnectionInterrupted()
-                        || !isHavingConnectivityIssues);
+                    isConnected,
+                    (isInLastN && isHavingConnectivityIssues)
```

Comment:
Isn't it better to leave only the isInLastN check here? If there aren't any connectivity issues the message won't be displayed anyway. It will be more clear for me.